presto/eval.py

```diff
@@ -50,12 +51,19 @@ def __init__(
         self,
         train_data: pd.DataFrame,
         val_data: pd.DataFrame,
+        predict_maize: bool = False,
```

Comment:
It seems this can quickly become chaotic. What if we want to map another crop?
Couldn't we work with a list specifying the target label(s) so we support any kind of crop we want?

If we do something like `predict_crops: List = [1200]`, that could be a start (though default value being a list is not a good idea if I recall well. But you get the point, would it make sense?

Reply:
Okay this makes sense to me. Added a `target_function` option to the eval task (and the dataset) instead of creating a new class.